src/brainCloudRttComms.js

```diff
@@ -233,14 +241,18 @@ function BrainCloudRttComms () {
      * Disables Real Time event for this session.
      */
     bcrtt.disableRTT = function() {
-        isEnabled = false;
+        bcrtt.isEnabled = false;
 
         if (bcrtt.heartbeatId) {
             clearInterval(bcrtt.heartbeatId);
             bcrtt.heartbeatId = null;
         }
 
         if (bcrtt.socket) {
+            bcrtt.socket.removeEventListener('error', bcrtt.onSocketError);
```

Comment:
Those 4 lines is the fix. Had to move those functions from lambdas to their own.